unit_tests/speech/test_client.py

```diff
@@ -0,0 +1,165 @@
+                "languageCode": "EN",
+                "profanityFilter": True
+            },
+            "audio": {
+                "content": _B64_AUDIO_CONTENT
+            }
+        }
+        credentials = _Credentials()
+        client = self._makeOne(credentials=credentials)
+        client.connection = _Connection(RETURNED)
+
+        encoding = Encoding.FLAC
+
+        response = client.sync_recognize(_AUDIO_CONTENT,
+                                         encoding,
+                                         self.SAMPLE_RATE,
+                                         language_code="EN",
+                                         max_alternatives=2,
+                                         profanity_filter=True,
+                                         speech_context=self.HINTS)
+
+        self.assertEqual(REQUEST,
+                         client.connection._requested[0]['data'])
+        self.assertEqual(response[0]["transcript"], 'hello')
+
+    def test_sync_recognize_without_optional_parameters(self):
+        from google.cloud.speech.client import Encoding
+        from unit_tests.speech._fixtures import SYNC_RECOGNIZE_RESPONSE
+
+        RETURNED = SYNC_RECOGNIZE_RESPONSE
+        REQUEST = {
+            "config": {
+                "encoding": "FLAC",
+                "sampleRate": 16000
+            },
+            "audio": {
+                "content": _B64_AUDIO_CONTENT
+            }
+        }
+        credentials = _Credentials()
+        client = self._makeOne(credentials=credentials)
+        client.connection = _Connection(RETURNED)
+
+        encoding = Encoding.FLAC
+
+        response = client.sync_recognize(_AUDIO_CONTENT,
+                                         encoding,
+                                         self.SAMPLE_RATE)
+
+        self.assertEqual(REQUEST,
+                         client.connection._requested[0]['data'])
+        self.assertEqual(response[0]["transcript"], 'hello')
+
+    def test_sync_recognize_without_content(self):
+        from google.cloud.speech.client import Encoding
+
+        credentials = _Credentials()
+        client = self._makeOne(credentials=credentials)
+
+        with self.assertRaises(ValueError):
+            client.sync_recognize(None, Encoding.FLAC, self.SAMPLE_RATE)
+
+    def test_sync_recognize_without_encoding(self):
+        credentials = _Credentials()
+        client = self._makeOne(credentials=credentials)
+
+        with self.assertRaises(ValueError):
+            client.sync_recognize(_AUDIO_CONTENT, None, self.SAMPLE_RATE)
+
+    def test_sync_recognize_without_samplerate(self):
+        from google.cloud.speech.client import Encoding
+
+        credentials = _Credentials()
+        client = self._makeOne(credentials=credentials)
+
+        with self.assertRaises(ValueError):
+            client.sync_recognize(_AUDIO_CONTENT, Encoding.FLAC, None)
+
+
+class _Credentials(object):
+
+    _scopes = ('https://www.googleapis.com/auth/cloud-platform')
```

Comment:
You're missing a comma in `('https://...')` to make it a tuple. Right now the parens don't do anything.